app/(playground)/p/[agentId]/actions.ts

```diff
@@ -118,7 +121,7 @@ export async function action(
 	agentId: AgentId,
 	nodeId: NodeId,
 ) {
-	const startTime = performance.now();
+	const startTime = Date.now();
```

Comment:
I'm not familiar with it, so please tell me, was Date.now more appropriate than Performance.now?

Reply:
I found `performance.now()` is not appropriate (completely different what we want) here because it returns a elapsed time from the Node.js process started @toyamarinyon 

FIY: the reason described on https://github.com/giselles-ai/giselle/pull/242/commits/a34c0252af4afaee6ccb0211edcaeb35a0fb7355